uni/lib/view/exams/widgets/exam_filter_form.dart

```diff
@@ -6,9 +6,12 @@ import 'package:uni/model/providers/lazy/exam_provider.dart';
 
 class ExamFilterForm extends StatefulWidget {
   const ExamFilterForm(this.filteredExamsTypes, {super.key});
```

Comment:
Now make this list an immutable view and create a local version of it in the state class.

Reply:
Isn't the widget.filteredExamTypes already a local version of the immutable view?